templates/zerver/help/linking-to-zulip-website.md

```diff
@@ -0,0 +1,33 @@
+# Linking to the Zulip website
+
+Zulip sponsors free [Zulip Cloud Standard](https://zulip.com/plans/) hosting for
+hundreds of worthy organizations. Zulip has also invested into making it as easy
+as possible to [self-host](https://zulip.com/self-hosting/) its [100%
+open-source](https://github.com/zulip/zulip#readme) software.
+
+If you would like to express your appreciation, and help others find out about
+Zulip, please list Zulip in the appropriate section of your organization's
+website (e.g., **Sponsors** or **Acknowledgements**).
```

Comment:
I think we kinda want 3 thoughts:

* We encourage, but do not require, organizations that Zulip sponsors to list Zulip in the appropriate section ...
* Here's how we prefer to be linked to.
* Perhaps: You can also support the Zulip project via GitHub Sponsors etc. Or maybe that could just be a "Related" link at the bottom.